### PR TITLE
Improve lighthouse cli

### PR DESCRIPTION
**Pull request by BeeGodwin**
- Rename package.json script for pretty-printing lighthouse jobs
- Call pretty-printed lighthouse job from CLI

### CODE REVIEW

File: lib/lighthouse.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const LIGHTHOUSE_OPTS = {
 function run() {
   const configName = getConfigName();
   const config = getConfig(configName);
-  const isTerser = getIsTerser();
+  const isPretty = process.env.A11Y_PRETTY === true || process.env.A11Y_PRETTY === 'true';
```
Comment (andymsuk) [inline lib/lighthouse.js:38]: Why does it need both?

Comment (BeeGodwin) [reply lib/lighthouse.js:38]: If it gets called from the package script, it'll be a string.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "start:bbc-a11y:ci": "npm run build:bbc-a11y && bash -c \"docker run --rm --tty --volume $PWD:/ws bbca11y/bbc-a11y-docker --config /ws/a11y.js --reporter /ws/lib/bbcA11YJUnitReporter.js; exit 0;\" && npm run clean",
     "start:lighthouse:junit": "node runLighthouse.js",
     "start:lighthouse:junit-headless": "A11Y_HEADLESS=true node runLighthouse.js",
-    "start:lighthouse:junit-headless-pretty": "A11Y_HEADLESS=true A11Y_TERSER=true node runLighthouse.js && yarn lighthouse-to-console",
+    "start:lighthouse:headless": "A11Y_HEADLESS=true A11Y_PRETTY=true node runLighthouse.js && npm run lighthouse-to-console",
```
Comment (andymsuk) [inline package.json:22]: I don't think `&& npm run lighthouse-to-console` is needed now that you manually call the console in the code.

File: lib/lighthouse.js (hunk 1 above)
Comment (andymsuk) [reply lib/lighthouse.js:38]: Just use a string everywhere then? :)

Comment (BeeGodwin) [reply lib/lighthouse.js:38]: lol ok- it means putting a `thing = 'true'` right next to an `otherThing = true` which kinda crimped my sense of orderliness 😂

**Pull request merged**